### ISSUE TITLE
bower EINVALID; The "main" field has to contain only 1 file per filetype

### ISSUE THREAD

**Issue by frankcortes** [rollbar/rollbar.js#126]
When we are trying to install `rollbar` (with `bower = 1.4.1`) with the command `bower install rollbar`, shit happens. The following error is shown:

```bash
 ** bower EINVALID      Failed to read /tmp/deploy/bower/rollbar-760-ETgKQf/bower.json
 ** Additional error details:
 ** The "main" field has to contain only 1 file per filetype; found multiple .js files: 
["dist/rollbar.umd.js","dist/rollbar.snippet.js","src/plugins/jquery.js"]
```
PD: We are using bower in a computer with Debian. I am only able to reproduce this bug in a computer which uses this OS. Otherwise, the `bower` command is working fine.

```bash
bash>$ cat /etc/issue
Debian GNU/Linux 7 \n \l
```

**Comment by coryvirok** [comment]
Thanks for the report, @frankcortes. I'll add it to our sprint to look at soon.

**Comment by brianr** [comment]
@frankcortes What version of Debian is this on? Are you still having this problem with the latest version of rollbar?

I was just able to successfully install 1.6.0 on an Ubuntu 12.04 machine, using bower 1.4.1:

```
$ bower install rollbar 
bower cached        git://github.com/rollbar/rollbar.js.git#0.10.10
bower validate      0.10.10 against git://github.com/rollbar/rollbar.js.git#*
bower new           version for git://github.com/rollbar/rollbar.js.git#*
bower resolve       git://github.com/rollbar/rollbar.js.git#*
bower download      https://github.com/rollbar/rollbar.js/archive/v1.6.0.tar.gz
bower extract       rollbar#* archive.tar.gz
bower resolved      git://github.com/rollbar/rollbar.js.git#1.6.0
bower install       rollbar#1.6.0

rollbar#1.6.0 bower_components/rollbar


┌──────────────────────────────────────────┐
│ Update available: 1.5.1 (current: 1.4.1) │
│ Run npm install -g bower to update.      │
└──────────────────────────────────────────┘

$ cat /etc/issue
Ubuntu 12.04.5 LTS \n \l
```

**Comment by frankcortes** [comment]
Using the last version (`1.6.0`) the issue still persists:
```
$ bower install rollbar
bower rollbar#*             not-cached git://github.com/rollbar/rollbar.js.git#*
bower rollbar#*                resolve git://github.com/rollbar/rollbar.js.git#*
bower rollbar#*               download https://github.com/rollbar/rollbar.js/archive/v1.6.0.tar.gz
bower rollbar#*                extract archive.tar.gz
bower rollbar#*               EINVALID Failed to read /tmp/deploy/bower/rollbar-3404-zcVk7W/bower.json

Additional error details:
The "main" field has to contain only 1 file per filetype; found multiple .js files: ["dist/rollbar.umd.js","dist/rollbar.snippet.js","src/plugins/jquery.js"]
```

**Comment by coryvirok** [comment]
Fixed in v1.6.1 - https://github.com/rollbar/rollbar.js/blob/v1.6.1/bower.json

**Comment by brianr** [comment]
@frankcortes Is this working for you now?
